src/main/pages/che-7/end-user-guide/proc_creating-a-workspace-from-with-a-publicly-accessible-standalone-devfile-using-http.adoc

```diff
@@ -0,0 +1,25 @@
+// configuring-a-workspace-using-a-devfile
+
+[id="creating-a-workspace-from-with-a-publicly-accessible-standalone-devfile-using-http_{context}"]
+= Creating a workspace from/with a publicly accessible standalone devfile using HTTP
+
+A workspace can be created using a devfile, the URL of which is pointing to the raw content of the devfile. The Che 7 instance then uses the discovered devfile to build a workspace.
+
+.Prerequisites
+* A running instance of Eclipse Che. To install an instance of Eclipse Che, see link:{site-baseurl}che-7/che-quick-starts/[Che 'quick-starts'].
+* The publicly-accessible standalone `devfile.yaml` file. See link:{site-baseurl}che-7//making-a-workspace-portable-using-a-devfile/[Making a workspace portable using a Devfile] for detailed information about creating and using devfiles.
+
+.Procedure
+. Execute the workspace by opening the following URL:
++
+[subs="+quotes"]
+----
+https://__<your-che-host>__/f?url=https://<yourhosturl>/devfile.yaml
+----
++
+Example:
++
```

Comment:
@MichalMaler, fix this section in the same way I suggested for the two similar sections above.

Reply:
Elegant solution! Thank You!